test/e2e/e2e_suite_test.go

```diff
@@ -174,3 +187,21 @@ func getMCPUpdateValueFromEnv(envVar string, fallback time.Duration) time.Durati
 	Expect(err).NotTo(HaveOccurred())
 	return timeout
 }
+
+func pauseMCPs(nodeGroups []nropv1alpha1.NodeGroup, pause bool) {
+	mcps, err := controllers.GetNodeGroupsMCPs(context.TODO(), e2eclient.Client, nodeGroups)
+	if err != nil {
+		klog.Warningf("failed to get the MCPs resources: %v", err)
+		return
```

Comment:
on second thought we should `ExpectNoError` here. Failing to get this information is worth aborting the test. Can be fixed later.